Mathlib/Analysis/InnerProductSpace/WeakOperatorTopology.lean

```diff
@@ -0,0 +1,49 @@
+/-
+Copyright (c) 2024 Frédéric Dupuis. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Frédéric Dupuis
+-/
+
+import Mathlib.Analysis.InnerProductSpace.Dual
+import Mathlib.Analysis.NormedSpace.WeakOperatorTopology
+
+/-!
+# The weak operator topology in Hilbert spaces
+
+This file gives a few properties of the weak operator topology that are specific to operators on
+Hilbert spaces. This mostly involves using the Fréchet-Riesz representation to convert between
+applications of elements of the dual and inner products with vectors in the space.
+-/
+
+open scoped Topology
+
+namespace ContinuousLinearMapWOT
+
+variable {𝕜 : Type*} {E : Type*} {F : Type*} [RCLike 𝕜] [AddCommGroup E] [TopologicalSpace E]
+  [Module 𝕜 E] [NormedAddCommGroup F] [InnerProductSpace 𝕜 F]
+
+lemma ext_inner_iff {A B : E →WOT[𝕜] F} : A = B ↔ ∀ x y, ⟪y, A x⟫_𝕜 = ⟪y, B x⟫_𝕜 := by
```

Comment:
I think we may also want `ext_inner_iff.mpr` to be an `@[ext]` lemma.

Reply:
Does `ext` behave well where there are competing `ext` lemmas? If we have both this and `ext_dual_iff.mpr` tagged as `ext`, they would both apply in the Hilbert space case.

Reply:
I think this was a problem at one point, but I don't think it is any longer. I believe we have resurrected the Lean 3 behavior where the one declared later has priority, so in this case, the inner product version. You could test it though.

Reply:
I had a quick look at the `ext` code, and it uses a discrimination tree (as I assumed), so the order of declaration probably doesn't matter. However, we can set priorities on ext lemmas so I'll just do that.